packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -88,7 +88,7 @@ export const createSeatManager = (
     return activeZCFSeats.get(zcfSeat);
   };
 
-  const ZCFSeatI = M.interface('ZCFSeat', {}, { sloppy: true });
+  const ZCFSeatI = M.interface('ZCFSeat', {}, { defaultGuards: 'passable' });
```

Comment:
This strikes me as a particular dangerous place to omit explicit method guards. Am I missing something?